src/components/cloud-cover/demo/content.twig

```diff
@@ -5,7 +5,7 @@
     } only %}
       {% block content %}
         {% include '@cloudfour/components/heading/heading.twig' with {
-          level: -2,
+          level: -1,
```

Comment:
This change makes me wonder if there aren't other unexpected changes that might arise from this. For example, would buttons grow unexpectedly large as well?

Maybe the font size deserves its own class to better confine it. Or even some sort of `u-text-grow` utility?

Curious if anyone in @cloudfour/dev has an opinion (especially @spaceninja since he's usually my "utility or element class" conscience).

Reply:
In this case, if the goal is to only affect the text size of the "description" content of the Cloud Cover header, then I'd say target the change to just that element (either via utility class or BEM class).

If the goal is to increase the text size of the header _overall_, including the heading and any other elements that might appear, such as a signup form, then this approach seems reasonable to me.

As for the "utility vs element class" argument, you caught the wrong week to ask me — have just done a LOT of manual search-and-replace of utility text colors in a client project, I'm feeling a bit down on utilities at the moment. I think they're great for prototyping, but as a broad rule of thumb, I'm finding myself swinging back in favor of modular CSS approaches like BEM — so that pattern library updates only need to happen in the CSS layer, and design changes are less likely to trigger a breaking change in the markup layer. But that's probably a bigger discussion than this PR, and since we're using utilities elsewhere, I wouldn't be mad to see them used to solve this problem.